docs/sources/developers/plugins/backend/debug-your-backend-plugin.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Debug your backend plugin"
+keywords = ["grafana", "plugins", "backend", "plugin", "backend-plugins", "documentation"]
+type = "docs"
++++
+
+# Debug your backend plugin
+
+This guide explains how to attach a debugger to a running backend plugin.
+
+A backend plugin runs in its own process, separate from grafana-server. This makes it difficult to debug the plugin the same way you would debug the grafana-server process.
+
+You can read more about the how plugin processes are managed in the [go-plugin](https://github.com/hashicorp/go-plugin#architecture) project page.
+
+Fortunately, you can debug your plugin by attaching the [Delve](https://github.com/go-delve/delve) debugger for Go to the subprocess running your plugin.
+
+You can add more information available while debugging by building your plugin with additional flags.
```

Comment:
Add an h2 here describing what this process is. This maintains consistency with the following h2 section and makes it easier for users to understand the document. In tech writing, we often call that a "signpost."